apps/mull-api/src/app/event/event.service.spec.ts

```diff
@@ -186,4 +187,9 @@ describe('EventService', () => {
     const image = await service.getEventImage(36);
     expect(image).toEqual(mockAllEvents[1].image);
   });
+
+  it(`should return a user's portfolio`, async () => {
+    const portfolio = await service.getUserEventsPortfolio(1);
```

Comment:
Minor, just makes the test easier to read since there's no magic number
```suggestion
    const portfolio = await service.getUserEventsPortfolio(mockAllEvents[0].host.id);
```